src/button/button.style.tsx

```diff
@@ -21,6 +21,10 @@ export const Main = styled.button<MainStyleProps>`
         }
     }
 
+    ${MediaQuery.MaxWidth.mobileM} {
+        width: 100%;
+    }
```

Comment:
the reason why I did this was to make sure that the button is shifted down on mobile and it would occupy the full width of the respective container. but I'm not sure about the dimension for the mobile

Reply:
have clarified with UX that the mobile layout should apply on mobileL now